### ISSUE TITLE
2.6.1 First time I opened the Language choices option I only saw 3 choices

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#979]
The "drop-down" menu option for the UI Language selector acted weird on first use.
The first time I clicked on "Language (English)" button, the option menu only contained the 1st THREE language options. 
![image](https://github.com/user-attachments/assets/f3b5518f-a3a1-438f-9405-ffbc99be4b45)

I had to use the keyboard down arrow key to get the list to scroll down to Indonesian (or any of the other languages below "English"). 

After changing to Indonesian (which restarted PTXprint and worked fine) and back to English (which restarted PTXprint and worked fine too), the menu has now populated itself and displays ALL available UI language options. 

I had just updated to v2.6.1, so maybe the list needed to be forced to refresh? 

This issue would be a problem for mouse users, as there is no way to scroll the truncated menu with the mouse. And there is no indication in the truncated option menu that there are more choices below "English".

**Comment by markpenny** [comment]
I confirm that I have also seen this bug from time to time (on Windows). But at this point I haven't found a predictable way of repeating it, let alone a way to solve it. But we'll keep an eye on it and keep looking for a solution.

**Comment by mhosken** [comment]
We could add some telemetry (debug logging) and then that can help in situe debugging.

**Comment by mhosken** [comment]
Please run debug if you encounter this and send in a ptxprint.log

**Comment by davidc86** [comment]
When it shows up, there is no time to "run debug". It just did it again in v2.6.8 (the first time I tried to open that menu), so I came here it try and figure out what you meant by "run debug" (which is still not clear), and when I switched back to PTXprint the dialog opened normally. So that is the span of a few seconds. Maybe it is only the very first time that menu is opened that the list is not fully populated? But the subsequent times it is?

Sorry I couldn't help. And sorry I don't know how to "run debug" on PTXprint.

**Comment by davidc86** [comment]
This happened again with version 2.7.13

First time I clicked on the option: 

![Image](https://github.com/user-attachments/assets/63e10060-2580-4b6c-a825-86663ceafd62)

It resolved itself after I clicked away and came back to the option. 

Strange

**Comment by markpenny** [comment]
Yes, I've seen this myself a few times, and don't yet know what is causing it, or how to reliably replicate it in order to fix it. One day we'll catch this elusive little fox!
